frontend/src/data/variables/AcsHealthInsuranceProvider.ts

```diff
@@ -77,6 +77,14 @@ class AcsHealthInsuranceProvider extends VariableProvider {
       );
     }
 
+    //Remove white hispanic to bring inline with others
```

Comment:
Consider linking to the issue (https://github.com/SatcherInstitute/health-equity-tracker/issues/594)  in the comment here as a way for readers to see more detail